### PR TITLE
[Docs] Fix typos and minor grammar mistakes

### PR DESCRIPTION
**Pull request by glenacota**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? ✅ 
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)? ✅

### CODE REVIEW

File: docs/painless/painless-guide/painless-datetime.asciidoc [hunk 1]
```diff
@@ -600,18 +600,18 @@ if (doc.containsKey('start') && doc.containsKey('end')) { <1>
 contain a specific field. Use the `containsKey` method call on the `doc` input
 to ensure a field exists as part of the index for the current document.
 <2> Some fields within a document may have no values. Use the `size` method
-call on a field within the `doc` input to ensure that field has at least one
+call on a field within the `doc` input to ensure that a field has at least one
```
Comment (jrodewig) [inline docs/painless/painless-guide/painless-datetime.asciidoc]: I believe the previous change was a bit more clear.

`that field` refers to the field on which the `size` method was called. Here's my recommendation:

> call on a field within the `doc` input to ensure the field has at least one

**Review by jrodewig — CHANGES_REQUESTED** [review]
Looks good overall. I left a comment for one of the suggested changes.

After that is addressed, I can merge this and port to other branches. Thank you @glenacota.

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Comment by jrodewig** [comment]
Hi @glenacota. Do you plan to address the feedback listed here?

If not, I'll close this PR and treat it as a bug report.

Thanks again for your contribution.

**Comment by jrodewig** [comment]
Closing due to lack of response. Treating as a bug report.

**Comment by glenacota** [comment]
Apologies for disappearing, and thanks for having taken care of it.

**Pull request closed without merging**